### PR TITLE
FINERACT-1047 Added integration test for limit param (audit trails)

### PR DESCRIPTION
**Pull request by thesmallstar**
## Description
This checks if audits with limit=5(for example) is called, we receive exactly 5 audits from server.

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [x] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [x] API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm has been updated with details of any API changes.

- [x] Integration tests have been created/updated for verifying the changes made.

- [x] All Integrations tests are passing with the new commits.

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

**Comment by thesmallstar** [comment]
this will have merge conflict with #1165  depending upon which is merged first.

File: fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java [hunk 1]
```diff
@@ -102,4 +103,29 @@ public void auditShouldbeCreated() {
         auditHelper.verifyOneAuditOnly(auditsRecieved, officeId, "CREATE", "OFFICE");
     }
 
+    /**
+     *
```
Comment (vorburger) [inline fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java:107]: Remove such empty comment lines in the future

File: fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java [hunk 2]
```diff
@@ -102,4 +103,29 @@ public void auditShouldbeCreated() {
...
+     * Here we test that audit request with limit x only returns x entries
```
Comment (vorburger) [inline fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java:108]: Just use "Tests that..." language in the future, or -even better- just use "Tests that..." language in the future, or -even better- use a method name saying what you document - it's OK for tests to have long names.. :-)

**Review by vorburger — APPROVED** [review]

File: fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java (hunk 2 above)
Comment (thesmallstar) [reply fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java:108]: fixing in #1164

File: fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java (hunk 1 above)
Comment (thesmallstar) [reply fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AuditIntegrationTest.java:107]: fixing in #1164

**Pull request merged**